api/shared-modules/src/node/utils/encryption/index.ts

```diff
@@ -48,3 +49,21 @@ export const decrypt = (cipher: string, secret: string) => {
 	const decrpyted = Buffer.concat([decipher.update(Buffer.from(hash, 'hex')), decipher.final()]);
 	return decrpyted.toString();
 };
+
+export const asymEncrypt = (data: any, privateKey: string, channelPublicKey: string): string => {
```

Comment:
can you please name it public key instead of `channelPublicKey` to have it more generic also for asymDecrypt